metric/system/process/process.go

```diff
@@ -280,6 +280,7 @@ func (procStats *Stats) pidFill(pid int, filter bool) (ProcState, bool, error) {
 
 	// network data
 	if procStats.EnableNetwork {
+		// TODO: this can't handle hostfs settings
```

Comment:
Does this TODO still need to be done? Or can this comment be removed?

Reply:
...I need to file a bug for it, thanks for reminding me